DANS-KNAW/easy-deposit
# EASY-1145  Using bag-store in easy-sword2 configurable

fixes EASY-1145

#### When applied it will
* it will check that either both bag-store `base-dir` and `base-url` are given, or that neither of them is given
* it will skip checking existence of the `bag-store ` `base-dir` if `base-dir` and `base-url` are absent
* if both bag-store base-dir and base-url are absent, all fetch-txt items will be fetched (also those referring to the bag-store)

#### How should this be manually tested?
* set bag-store.base-dir and bag-store.base-uri to an empty string, or just one of them to an empty string.
* send a bag to sword2 and check the status

@DANS-KNAW/easy
